postgres/table.go

```diff
@@ -13,6 +13,9 @@ type readableTable interface {
 	// Generates a select query on the current tableName.
 	SELECT(projection Projection, projections ...Projection) SelectStatement
 
+	// Creates a Lateral query on the current tableName.
+	LATERAL(table SelectTable) ReadableTable
```

Comment:
Why?

Reply:
To be used as shortcut, so it can be used like:

```go
Users.SELECT(Users.AllColumns()).FROM(Users.LATERAL(subquery...))

Users.LATERAL(subquery...).SELECT(...)
```

It can be eliminated if required.